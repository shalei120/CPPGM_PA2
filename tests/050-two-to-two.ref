invalid 2.2.2
eof
