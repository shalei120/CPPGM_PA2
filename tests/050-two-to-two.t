// http://forum.cppgm.org/question/738/suggestions-for-some-more-tests/?answer=1510#post-id-1510
// jdv
2.2.2
