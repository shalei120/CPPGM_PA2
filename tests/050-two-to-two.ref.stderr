ERROR: invalid floating literal (#1): 2.2.2
